pkg/commands/deploy.go

```diff
@@ -156,4 +147,17 @@ func deployOnCluster(commonOpts *CommonOptions, opts *deployOptions) error {
 		return err
 	}
 	return nil
+}
+
+func removeFromCluster(commonOpts *CommonOptions, opts *deployOptions) error {
```

Comment:
but this is used in just one place - `deployOnCluster` is used in two places, so is it really needed?